decidim-core/app/controllers/decidim/account_controller.rb

```diff
@@ -18,7 +18,7 @@ def update
       @account = form(AccountForm).from_params(params)
 
       UpdateAccount.call(current_user, @account) do
-        on(:ok) do |_account, unconfirmed_email|
+        on(:ok) do |unconfirmed_email|
```

Comment:
`UpdateAccount` is broadcasting a boolean. `unconfirmed_email` seems like a string to me. Can you change this to `email_is_unconfirmed` or something that makes sense?

Or maybe, instead of broadcasting a boolean, we can actually broadcast the email. We'd need to change the command, line 25, to `broadcast(:ok, @user.unconfirmed_email.presence)`

https://github.com/decidim/decidim/blob/3f392af2e2f5d8683225a629fd87c32c1e9c4989/decidim-core/app/commands/decidim/update_account.rb#L25